app/aws/PasskeyDB.scala

```diff
@@ -0,0 +1,74 @@
+package aws
+
+import com.gu.googleauth.UserIdentity
+import com.webauthn4j.converter.AttestedCredentialDataConverter
+import com.webauthn4j.converter.util.ObjectConverter
+import com.webauthn4j.credential.CredentialRecord
+import com.webauthn4j.util.Base64UrlUtil
+import models.AwsCallException
+import software.amazon.awssdk.services.dynamodb.DynamoDbClient
+import software.amazon.awssdk.services.dynamodb.model._
+
+import scala.jdk.CollectionConverters._
+import scala.util.{Failure, Success, Try}
+
+object PasskeyDB {
+
+  private[aws] val tableName = "Passkeys"
+
+  private val credentialDataConverter = new AttestedCredentialDataConverter(
+    new ObjectConverter()
+  )
+
+  case class UserCredentialRecord(
```

Comment:
I'm not 100% sure this wrapper type carries its weight. Would it be easier for the rare cases where this is used to take two arguments? If this was a way to name amiguous parameters (e.g. two `String`s) it'd be a different story, and I'd say this was for sure useful.

Super marginal, whatever you think!

Reply:
I agree.  I think this is where I was going to do something more complicated with it but it doesn't have much value as it stands.

Reply:
Removed in 3b9e056ac405260285256ce8deead8b69bff6bd5